spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```

Comment:
When running Rspec, I am getting the following error:

Failure/Error:
  RSpec.describe UsersController, type: :controller do
  end

NameError:
  uninitialized constant UsersController
./spec/controllers/users_controller_spec.rb:3:in `<top (required)>'

This should be resolved before merging into master since it seems to be blocking all the other tests from running from what I can tell.